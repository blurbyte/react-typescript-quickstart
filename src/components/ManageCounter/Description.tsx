import styled from '@styles/styledComponents';
import MainParagraph from '../MainParagraph';

const Description = styled(MainParagraph)`
  flex: 1 0;
`;

export default Description;
