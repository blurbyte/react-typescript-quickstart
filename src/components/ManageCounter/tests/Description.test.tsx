import * as React from 'react';
import { shallow } from 'enzyme';

import Description from '../Description';

describe('Description', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<Description />);
    expect(wrapper).toMatchSnapshot();
  });
});
