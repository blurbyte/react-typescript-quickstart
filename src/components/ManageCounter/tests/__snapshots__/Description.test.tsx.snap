// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Description renders correctly 1`] = `
.c0 {
  -webkit-flex: 1 0;
  -ms-flex: 1 0;
  flex: 1 0;
}

<styled.p
  className="c0"
/>
`;
